example/cordova/www/pages/metrics.html

```diff
@@ -45,20 +45,31 @@ <h4>Response:</h4>
   return {
     beforeCreate() {
       var self = this;
-      self.$("#metrics-info").html("No metrics");
-      window.MobileCore.MetricsService.getMetrics(
-        function (metrics) {
+      self.$("#metrics-info").html("Fetching metrics...");
+
+      self.metricsService = new aerogear.MetricsService({ url: "http://192.168.1.6:3000/metrics" });
```

Comment:
We are not using config here. We should use config parser.

Reply:
True, I'll add it in the todo list